shared/test/integration/helpers/marionetteSearchTypes.js

```diff
@@ -0,0 +1,18 @@
+
+'use strict';
```

Comment:
Unused file. No selector has been defined thanks to that class. I'd use it, though. This will prevent magic strings all along the code.